shared/chat/inbox/index.desktop.js

```diff
@@ -18,17 +19,23 @@ import {virtualListMarks} from '../../local-debug'
 import {inboxWidth, getRowHeight} from './row/sizes'
 
 type State = {
+  isHovered: boolean,
   showFloating: boolean,
 }
 
 class Inbox extends React.PureComponent<Props, State> {
   state = {
+    isHovered: false,
```

Comment:
This being in `state` makes it kinda hard to use the gear menu:
![team-menu](https://user-images.githubusercontent.com/11968340/54540780-fa88a480-496e-11e9-8be7-0ef02a4b19f6.gif)
so either that should be fixed or maybe this can move out of `state` and be on the class.

Reply:
thanks for the suggestion, seemed to help.
:white_check_mark: